NetCoreBootstrap/Startup.cs

```diff
@@ -42,6 +47,25 @@ public void ConfigureServices(IServiceCollection services)
                 options.LoginPath = "/Account/Login";
                 options.AccessDeniedPath = "/Account/AccessDenied";
             });
+            JwtSecurityTokenHandler.DefaultInboundClaimTypeMap.Clear(); // => remove default claims

```

Comment:
hmm is this comment a TODO?

Reply:
Nope. It's a comment describing what the line does